src/sql/base/common/codicons.ts

```diff
@@ -13,6 +13,5 @@ export const enum SqlIconId {
 	addServerGroupAction = 'add-server-group-action',
 	addServerAction = 'add-server-action',
 	activeConnectionsAction = 'active-connections-action',
-	serverPage = 'server-page',
-	addAccountAction = 'add-linked-account-action'
```

Comment:
it is more than just removing the entry here, you also need to remove the css entry as well as the actual svg files